gemfiles/Gemfile-rails-6.0.x

```diff
@@ -2,7 +2,8 @@ source 'http://rubygems.org'
 
 gemspec path: '..'
 
-gem 'rails', '~> 6.0.0.beta3'
+gem 'rails', '~> 6.0.0.rc1'
+gem 'sqlite3', '>= 1.4'
```

Comment:
This may be better to change `adhoq.gemspec` and lock gemfiles files under Rails 5.1